pkg/fw/Dockerfile

```diff
@@ -1,6 +1,7 @@
-FROM lfedge/eve-alpine:9fb9b9cbf7d90066a70e4704d04a6fe248ff52bb as build
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
```

Comment:
is this to silence yetus?

Reply:
No, that's the frontend for the parser, so I can use ADD command with git support, etc.... https://docs.docker.com/build/dockerfile/frontend/